__tests__/mint-tokens.test.js

```diff
@@ -0,0 +1,66 @@
+import TestUtils from "./test-utils";
+
+describe("mint-tokens api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/mint-tokens")
+      .send({
+        token: "03",
+        amount: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
```

Comment:
I think you should validate you have an output with mint authority and another one with the 1 amount of the minted token.

Reply:
You are right, but testing it with a mocked fullnode is not a valid effort for now.